src/components/CandidateAvatar/CandidateAvatar.scss

```diff
@@ -0,0 +1,70 @@
+.CandidateAvatar {
+    position: relative;
+    display: inline-block;
+    height: 72px;
+    width: 72px;
+
+    &__ring {
+        width: 72px;
+        height: 72px;
+    }
+
+    &__circle {
+        fill: none;
+        stroke: var(--color-neutral);
+        stroke-dasharray: 0 999;
+        stroke-linecap: round;
+        shape-rendering: geometricPrecision;
+        transition: all .3s ease;
+        transform: rotate(-90deg);
+        transform-origin: 50% 50%;
+
+        &--context {
+            @each $context in (good, warning, bad) {
+                &_#{$context} {
+                    stroke: var(--color-#{$context});
+                }
+            }
+        }
+    }
+
+    &__percentage {
+        background-color: rgba(0,0,0,.25);
+        border-radius: 50%;
+        color: white;
+        cursor: default;
```

Comment:
what if the image is used as a link? This will overwrite the cursor. Maybe better not to set it explicitly?

Reply:
Removed it